src/file_management/core.clj

```diff
@@ -73,11 +75,15 @@
            - file-value {:value ;file content
                          :content-type }
    Returns: file url to s3"
-  (let [file-key (gets-file-key file-key)]
-    (s3/put-object credentials
-                   (:bucket credentials)
-                   file-key
-                   (:value file)
-                   {:content-type (:content-type file)}
-                   (s3/grant :all-users :read))
-    (get-file-url credentials file-key))))
+  (go
+    (let [file-key (gets-file-key file-key)]
+      (try
+        (s3/put-object credentials
```

Comment:
This put-object will block a core-async thread for a possible long time. I was wondering if this should use a [core.asyc/thread](https://clojure.github.io/core.async/#clojure.core.async/thread), and return that instead of the go block.